client/js/components/main/assessment.jsx

```diff
@@ -120,8 +117,8 @@ export class Assessment extends React.Component{
     let current = this.props.progress.currentItemIndex;
     let items = [];
     if(displayNum > 0 && displayNum < this.props.questionCount){
-      let start = current / displayNum;
-      let end = start + displayNum;
+      let start = parseInt(current / displayNum) * displayNum;
```

Comment:
Javascript you fiend! We need to parseInt so we don't get floating point numbers.